contentcuration/contentcuration/urls.py

```diff
@@ -184,6 +187,26 @@ def get_queryset(self):
         return AssessmentItem.objects.select_related('contentnode').filter(contentnode__tree_id__in=tree_ids).distinct()
 
 
+class TaskViewSet(viewsets.ModelViewSet):
+    queryset = Task.objects.all()
+    serializer_class = serializers.TaskSerializer
+    permission_classes = [permissions.IsAuthenticated]
+
+    # task creation and updates are handled by the Celery async task, so forbid them via API
+    def create(self, validated_data):
+        raise MethodNotAllowed('POST')
+
+    def update(self, *args, **kwargs):
+        raise MethodNotAllowed('PUT')
+
+    def perform_destroy(self, instance):
+        # TODO: Add logic to delete the Celery task.q
```

Comment:
Oooh what's `task.q` in here? It sounds like celery doesn't delete it automatically.

Reply:
Sorry, it's a typo task that you activate by being one key off when pressing tab and not realizing your cursor was still in the editor window, and then proceeding to hit save later. I'll just delete the q. :)

Reply:
Oh, and to answer your question about deleting the actual Celery task, we need to call revoke with `terminate=True` if we want to cancel it in Celery too. I avoided it for this phase because:

1) This is best tested once we have full integration support. We need to see how a task responds when we terminate it in the middle of touching the db, etc. We may have to write task-by-task termination logic.

2) We don't support canceling a task right now, so I figured we can add this later as a feature request. (We could actually just not implement DELETE for now entirely.)